api/src/main/java/io/opencensus/trace/export/RunningSpanStore.java

```diff
@@ -37,8 +37,19 @@
 @ThreadSafe
 public abstract class RunningSpanStore {
 
+  private static final RunningSpanStore NOOP_RUNNING_SPAN_STORE = new NoopRunningSpanStore();
+
   protected RunningSpanStore() {}
 
+  /**
+   * Returns the no-op implementation of the {@code RunningSpanStore}.
+   *
+   * @return the no-op implementation of the {@code RunningSpanStore}.
+   */
+  static RunningSpanStore getNoopRunningSpanStore() {
```

Comment:
Why do we sometimes use getNoop and sometimes newNoop we should be consistent.

Reply:
The logic here is that stateless no-ops are static; for stateful no-ops, **getNoop** will make user think it is static and that's why we rename it to newNoop.

Reply:
I think we can do better (probably in a new PR) and always do newXXX and cache them in the component.